src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -20,8 +20,7 @@ export class FeederPodcastModel extends BaseModel {
   authorEmail: string = '';
 
   VALIDATORS = {
-    link: [REQUIRED(), URL('Not a valid URL')],
-    newFeedUrl: [URL('Not a valid URL')]
```

Comment:
I'd agree that allowing someone to save a non-valid URL is a bad idea.  Saving a blank may be okay.